GLMakie/src/drawing_primitives.jl

```diff
@@ -274,34 +280,24 @@ function draw_atomic(screen::GLScreen, scene::Scene,
         # vector is triggered, but basically all internal objects use the vector of tuples version, and that triggers
         # both glyphcollection and position, so it still works
         if glyphcollection[] isa Makie.GlyphCollection
-            # here we lift the glyph collection
-            collect_glyph_data = (gcollection, projview, transfunc, pos, rotation, model, space, offset) -> begin
-                res = Vec2f0(widths(pixelarea(scene)[]))
-                preprojected_glyph_arrays(pos, gcollection, space, projview, res, offset, transfunc)
-            end
-            glyph_data = lift(collect_glyph_data, glyphcollection, scene.camera.projectionview, Makie.transform_func_obs(scene), args...)
+            # here we use the glyph collection observable directly
+            gcollection = glyphcollection
         else
-            # and here we don't because it triggers dimension mismatches
-            collect_glyph_data = (projview, transfunc, pos, rotation, model, space, offset) -> begin
-                gcollection = glyphcollection[]
-                res = Vec2f0(widths(pixelarea(scene)[]))
-                preprojected_glyph_arrays(pos, gcollection, space, projview, res, offset, transfunc)
-            end
-            glyph_data = lift(collect_glyph_data, scene.camera.projectionview, Makie.transform_func_obs(scene), args...)
+            # and here we wrap it into another observable
+            # so it doesn't triggers dimension mismatches
+            # the actual, new value gets then taken in the below lift with to_value
+            gcollection = Observable(glyphcollection)
```

Comment:
why Observable{Observable{Vector{GlyphCollection}}}?

Reply:
So one can pass the observable through `lift` ... This avoids using two different functions for those code paths